types.d.ts

```diff
@@ -5579,6 +5579,11 @@ declare interface JavascriptParserOptions {
 	 */
 	createRequire?: string | boolean;
 
+	/**
+	 * Specifies global fetchPriority for dynamic import.
```

Comment:
@aboktor , I think it would be helpful to note the default value is "auto" (by which I assume it means that we leave it up to the browser's defaults); and also what a boolean value would mean.

(Though like I mentioned in another comment, I'm finding it hard to understand what `true` or `false` would mean in this case at all; do we need them?)

Reply:
I agree, let's remove `true`, `false` requires to disable it, you can have a parent module with `webpackFetchPriority: "high"` and want to disable it for some deep modules, would say it's fine tuning, we use the same logic for any magic comments, also some libraries can use `webpackFetchPriority` comment too but you don't want it, so we allow to disable it